addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -1,12 +1,216 @@
+                wizard.enable_send_mail = False
+                wizard.checkbox_send_mail = False
+                wizard.send_mail_readonly = False
+                wizard.send_mail_warning_message = None
+
+    @api.depends('move_ids', 'enable_l10n_it_edi')
+    def _compute_enable_send_by_post(self):
+        # EXTENDS snailmail_account
+        super()._compute_enable_send_by_post()
+        for wizard in self:
+            wizard.enable_send_by_post = wizard.enable_send_by_post and not (
+                wizard.enable_l10n_it_edi
+                and wizard.move_ids.filtered(lambda x: x._l10n_it_edi_is_self_invoice())
+            )
+
+    @api.depends('move_ids')
+    def _compute_enable_l10n_it_edi(self):
+        italy = self.env.ref('base.it')
+        for wizard in self:
+            enable = False
+            for move in wizard.move_ids:
+                enable |= (
+                    move.company_id.account_fiscal_country_id == italy
+                    or move.journal_id.type == 'sale'
+                    or move.journal_id.type == 'purchase' and move._l10n_it_edi_is_self_invoice()
+                )
+            wizard.enable_l10n_it_edi = enable
+
+    @api.depends('move_ids')
+    def _compute_l10n_it_edi_warning(self):
+        for wizard in self:
+            if not wizard.company_id.account_edi_proxy_client_ids.filtered(lambda x: x.proxy_type == 'l10n_it_edi'):
+                wizard.l10n_it_edi_warning = _("You must accept the terms and conditions in the Settings to use the IT EDI.")
+                continue
+            warnings = ""
+            for move in wizard.move_ids:
+                if (move_warnings := move._l10n_it_edi_export_data_check()):
+                    header = move.name + ":" if wizard.mode == 'invoice_multi' else None
+                    warnings += format_errors(header, move_warnings)
+            wizard.l10n_it_edi_warning = warnings
+
+    def _get_proxy_user(self):
+        self.ensure_one()
+        return self.company_id.account_edi_proxy_client_ids.filtered(lambda x: x.proxy_type == 'l10n_it_edi')
+
+    @api.depends('move_ids.journal_id', 'move_ids.invoice_pdf_report_id')
+    def _compute_l10n_it_edi_readonly(self):
+        """ Don't allow the checkbox to be clicked if there already is a PDF,
+            this may lead to inconsistencies between the PDF and the EDI XML
+        """
+        for wizard in self:
+            already_has_pdf = any(wizard.move_ids.mapped("invoice_pdf_report_id"))
+            already_has_xml = any([x._l10n_it_edi_check_processable() for x in wizard.move_ids.mapped("attachment_ids")])
+            wizard.l10n_it_edi_readonly = wizard.l10n_it_edi_warning or already_has_pdf or already_has_xml
+
+    @api.depends('l10n_it_edi_readonly', 'enable_l10n_it_edi')
+    def _compute_l10n_it_edi_checkbox(self):
+        for wizard in self:
+            wizard.l10n_it_edi_checkbox = wizard.enable_l10n_it_edi and not wizard.l10n_it_edi_readonly
```

Comment:
On multi mode, you can't use the checkbox. Suppose A, B, C 3 invoices but you need it EDI only on A. So the feature is there, same as the checkbox. Otherwise, you shouldn't do anything for B and C. That's why you need an extra method like `_get_default_enable_l10n_it_edi`.